model/src/guild/audit_log/change.rs

```diff
@@ -170,56 +217,71 @@ pub enum AuditLogChange {
     /// Whether emoticons are enabled.
     EnableEmoticons {
         /// Whether emoticons are now enabled.
-        #[serde(rename = "new_value")]
-        new: bool,
+        #[serde(rename = "new_value", skip_serializing_if = "Option::is_none")]
+        new: Option<bool>,
         /// Whether emoticons were enabled.
         #[serde(rename = "old_value", skip_serializing_if = "Option::is_none")]
         old: Option<bool>,
     },
+    /// Entity type of guild scheduled event was changed.
+    EntityType {
+        /// New entity type.
+        #[serde(rename = "new_value", skip_serializing_if = "Option::is_none")]
+        new: Option<bool>,
```

Comment:
Is't this not a bool? https://discord.com/developers/docs/resources/guild-scheduled-event#guild-scheduled-event-object-guild-scheduled-event-entity-types